app-misc/sl/files/Makefile-r4

```diff
@@ -0,0 +1,5 @@
+LDLIBS = `pkg-config --libs ncurses`
```

Comment:
Also, I think that that the current code would cause a build failure on systems having split ncurses, so there is no need for revbump (because nothing changes for people who built it, and others couldn't build it anyway).